res/skins/Tango/topbar.xml

```diff
@@ -140,7 +140,25 @@ Description:
         <SizePolicy>me,min</SizePolicy>
         <Children>
 
-          <WidgetGroup><Size>10me,1min</Size></WidgetGroup>
+          <StatusLight>
+            <TooltipId>FIXME</TooltipId>
```

Comment:
Maybe add a TODO to the PR description?

Reply:
Right, I actually wanted to fix that in this branch.
Funny thing is, even though it has a custom fixed tooltip the keyboard shortcut is appended:
![image](https://user-images.githubusercontent.com/5934199/98177467-fd952780-1efa-11eb-8e5b-a5a1cf2edb90.png)

Reply:
Done.
![image](https://user-images.githubusercontent.com/5934199/98295534-0137b580-1fb2-11eb-80f4-b86e851f8053.png)